Remove leading spaces of username when login

HuKeping opened the pull request:
When we run `docker login` we will be asked to input our username.

Prior to this PR, "userA" and " userA" will be considered as two
diferrent user, which is not right.

Signed-off-by: Hu Keping hukeping@huawei.com

LK4D4 commented:
@HuKeping Actually hub is accepting logins with leading spaces :) But I'm with you that that's not cool.

crosbymichael commented:
LGTM

tiborvass commented:
LGTM

HuKeping committed:
Remove leading spaces of username when login

When we run `docker login` we will be asked to input our username.

Prior to this PR, "userA" and " userA" will be considered as two
diferrent user, which is not right.

Signed-off-by: Hu Keping <hukeping@huawei.com>
